packages/@angular/cli/models/command.ts

```diff
@@ -14,16 +14,19 @@ export enum CommandScope {
 }
 
 export abstract class Command {
-  constructor(context: CommandContext, logger: Logger) {
+  constructor(context: CommandContext, logger: logging.Logger) {
     this.logger = logger;
     if (context) {
       this.project = context.project;
       this.ui = context.ui;
     }
   }
 
-  initialize(_options: any): Promise<void> {
-    return Promise.resolve();
+  async initializeRaw(args: any): Promise<any> {
```

Comment:
Like the earlier comment, I think this implementation should preserve the raw options on the base class as a protected property.

Reply:
done.